index.html

```diff
@@ -4021,11 +4021,14 @@ <h2>
 DID URL Dereferencing
       </h2>
       <p>
-The DID URL dereferencing function dereferences a <a>DID URL</a> into
+The <a>DID URL dereferencing</a> function dereferences a <a>DID URL</a> into
 a resource with contents depending on the <a>DID URL</a>'s components,
 including the <a>DID method</a>, method-specific identifier, path, query,
 and fragment. This process depends on <a>DID resolution</a> of the <a>DID</a>
-contained in the <a>DID URL</a>.
+contained in the <a>DID URL</a>. <a>DID URL dereferencing</a> might involve
+multiple steps (e.g., when the DID URL being dereferenced includes a fragment),
```

Comment:
Are there any other dereferencing libraries in existence that do this? This might confuse people that are used to their resolution libraries not doing this part of the process for them.

Reply:
Nobody is saying that the entire dereferencing process has to be implemented in a single library. As we know, in the case of HTTP URLs, the dereferencing processes involves both server and client components. In the case of DID URLs, those details depend on the method and concrete dereferencer implementation. This PR just clarifies that dereferencing returns the final resource after all the steps (including potential "client"-side fragment dereferencing) are completed.

Reply:
Hmm, perhaps we could use a less misleading example then... like: "e.g., when the DID URL being dereferenced requires the resolution process to query a Decentralized Ledger."

Reply:
Waiting on @peacekeeper to respond to see if he's ok w/ this change before merging.

Reply:
@msporny the intent of this PR was to address https://github.com/w3c/did-core/issues/399, which specifically asked the question whether fragment processing is part of resolving/dereferencing or not.

Maybe @iherman could comment if this PR is helpful, or if we can do something else to address his original issue.